### PR TITLE
Fix 01656_test_query_log_factories_info with analyzer.

### PR DESCRIPTION
**Pull request by KochetovNicolai**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
*This is an automated comment for commit 819effb6db1db310a0644292277ebf9bdcc7472e with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/60037/819effb6db1db310a0644292277ebf9bdcc7472e/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/60037/819effb6db1db310a0644292277ebf9bdcc7472e/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: tests/analyzer_tech_debt.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-00223_shard_distributed_aggregation_memory_efficient
```
Comment (novikd) [inline tests/analyzer_tech_debt.txt:1]: Seems like you forgot to remove the correct test from the list.

**Review by novikd — COMMENTED** [review]
In general LGTM

**Review by alexey-milovidov — APPROVED** [review]

**Comment by tavplubix** [comment]
@KochetovNicolai, you forgot to check the CI failures before merging. This PR breaks `00223_shard_distributed_aggregation_memory_efficient` with analyzer

**Pull request merged**